spanner/spanner_snippets/snippet.go

```diff
@@ -1266,6 +1277,329 @@ func updateUsingBatchDML(ctx context.Context, w io.Writer, client *spanner.Clien
 
 // [END spanner_dml_batch_update]
 
+// [START spanner_create_table_with_datatypes]
+
+func createTableWithDatatypes(ctx context.Context, w io.Writer, adminClient *database.DatabaseAdminClient, database string) error {
```

Comment:
Please initialize clients in the body of the sample, not passed as an argument.

https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#initialize-clients-and-services-in-every-sample

Reply:
This matches the sample's existing style. This can be revisited in a future update.